data/talkactions/scripts/start_cast.lua

```diff
@@ -0,0 +1,9 @@
+function onSay(cid, words, param)
+	local player = Player(cid)
```

Comment:
Reason is most likely since the system was added before or not long after TFS started to push userdata.